.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
+max-line-length = 79
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
```

Comment:
Don't forget to include empty lines at the end of your files. You can configure VS Code to add these automatically.